### DISCUSSION TITLE
2초 패치이후에 잡으신분 계신가요 ... ?

### DISCUSSION THREAD

**Discussion by chopinballadeno4** [changdoc/naver-vaccine-macro#68] [Q&A]
기여해주시는 개발자님들 감사합니다 .. ㅠㅠ

2초 1개병원 패치 이후에 병원 3트정도 했었는데 빰빠래 울리고
예약시도 하는거 같은데 다 예약 실패로 뜨네요 ..

**Comment by OneTop4458** [comment]
네 확인된 결과 3분 넘게 성공하신 것으로 파악됩니다.
이게 지역 및 병원에 따라 캐바캐가 심합니다.

예를 들어 개발자가 많은 가산 지역의 경우 실패 확률이 매우 높은데
뭐 연신내 인천 이런 쪽은 1트 만에 다들 성공하시더라고요..

병원을 바꿔보시길 추천 드립니다.

**Reply by chaeun9811** [reply]
가산지역이 원래힘들었군요,, 저도 오늘 가산지역에서 실패했습니다.. 카톡알림보다 빵빠레가 먼저울렸는데 예약실패더군요 ,,

**Reply by ghost** [reply]
저 오늘 깔끔하게 1트만에 클리어 했습니다 ㅜㅜ 정말 감사합니다 개발자횽님.. 다들 성공하실거에요! 믿고 기다려 보세용 화이팅

**Reply by chopinballadeno4** [reply]
정보 공유 감사합니다 !

**Comment by soosooho** [comment]
어제 한번에 제꺼 성공하고 가족꺼 해주는데 한번 실패했습니다.

**Reply by chopinballadeno4** [reply]
정보 공유 감사합니다 !

**Comment by baruegen** [comment]
오늘 부천 1개, 대전 1개 잡았습니다.

**Reply by taeheelee5** [reply]
부천 혹시 어디가 잘뜨는 병원인지 알려주실수있나요?? 부탁드립니다 저도 맞고싶어요ㅠㅠㅠ

**Reply by seokwonchoisnu** [reply]
저는 계속 실패하네요 동작구 쪽인데 2초의 텀이 있어서 그런지 예약실패를 2번했네요... 당장 5주후쯤 해외에 나가봐야해서.. 차타고 가서 맞으면 되는데  부천이나 인천쪽은 잘잡히나요? 보통 1~2시간안에 되기도하나요?

**Reply by baruegen** [reply]
김가정의학과의원에서 잡았는데 아침부터 쭉 하다가 17시쯤인가 안됐을 때 잡혔던거 같습니다

**Reply by cherryoneshot** [reply]
혹시 대전지역은 어디가 잘 뜨나요?

**Comment by OneTop4458** [comment]
한 가지 팁을 드리자면
매크로 써도 절때 예약 못 할 듯한 병원이 존재 합니다..
> 왜냐 큰 병원이나 사람이 몰릴만한 병원은 노리고 있는 사람이 크다 = 경쟁자가 많다 = 실패 확률 UP

예를 들어 가산 기준 
"사단법인대한산업보건협회부설서울의원" 의 경우 매크로로 아는 지인 포함 많이 돌렸지만 성공률 0% 입니다..
반면 오히려 작은 내과 의원급들은 알림 거는 사람도 적고 그러니 1트만에 성공활 확률이 높구요

**Comment by seokwonchoisnu** [comment]
동작구, 강남구에서 작은 내과 위주로 3일째 하고 있는데 오늘 오전까지 6번 실패 했어요. 다시 해보는 중입니다.

**Comment by IAmEnchant** [comment]
오늘 강남구 성공했습니다.
